CONTRIBUTING.rst

```diff
@@ -879,7 +879,7 @@ Apache Airflow is a Community within Apache Software Foundation. As the motto of
 the Apache Software Foundation states "Community over Code" - people in the
 community are far more important than their contribution.
 
-This means that communication plays a big role in it, and this chapter is all about it.
+This means that communication plays a big role in it, and this chapter is all about it. This is really important.
```

Comment:
```suggestion
This means that communication plays a big role in it, and this chapter is all about it. 
```

Reply:
This sentence is strong enough. WDYT? In addition, it is in a separate paragraph.